src/components/draft-js-editor/__tests__/DraftJSEditor.test.js

```diff
@@ -66,7 +66,7 @@ describe('components/draft-js-editor/DraftJSEditor', () => {
                 'aria-autocomplete': 'list',
                 'aria-expanded': true,
                 'aria-owns': 'id',
-                role: 'combobox',
+                role: 'textbox',
```

Comment:
So we're still using `inputProps.textbox` which I don't think is what we're referencing above: https://github.com/box/box-ui-elements/pull/2639/files#diff-e198f1f72d6edc0d491d8f2b2c7050db6bf462561b0e533630439673f064717aR114

Reply:
Also, the `a11yProps` are wrapped in an `if` condition if the `inputProps.role` is set: https://github.com/box/box-ui-elements/blob/master/src/components/draft-js-editor/DraftJSEditor.js#L108-L116

Since we no longer care/require that prop, should we remove the conditional?